src/meta.rst

```diff
@@ -186,7 +186,7 @@ while adding ``pip`` to the build requirements:
         - pip
 
 These options should be used to ensure a clean installation of the package without its
-dependencies and without ``egg-info`` directories, which do not interact well with conda.
+dependencies, which do not interact well with conda.
```

Comment:
"which do not interact well with conda" should be applicable to both, `egg-info` dirs and `pip`-installed deps, so I left this in. @nicoddemus, would you agree this is fine?

Reply:
Sure, sounds fine to me! 👍